src/lib.rs

```diff
@@ -311,3 +319,48 @@ impl NixPath for PathBuf {
         self.as_os_str().with_nix_path(f)
     }
 }
+
+// APPENDIX
```

Comment:
Scootched to the PR description. I guess I'm worried it might get lost, but it took like 15 mins to write so that wouldn't be the end of the world. :)